### PR TITLE
navbar fix, seo pages added, faq fix

### PR DESCRIPTION
**Pull request by alias01**

### CODE REVIEW

File: src/router/routes.js [hunk 1]
```diff
@@ -128,6 +128,15 @@ export const routes = [
       title: "Parking near "
     }
   },
+  // Todo Delete below code before deployment
```
Comment (arrogant-ant) [inline src/router/routes.js:131]: please delete this code

File: src/store/index.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import Vue from 'vue'
+import Vuex from 'vuex'
+
+Vue.use(Vuex)
+
+const store = new Vuex.Store({
+    state: {
+        count: 10
```
Comment (arrogant-ant) [inline src/store/index.js:8]: please add a comment stating the significance of 10

File: src/views/temp.vue [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<template>
```
Comment (arrogant-ant) [inline src/views/temp.vue:1]: please check if we can get rid of this file

**Review by arrogant-ant — CHANGES_REQUESTED** [review]

File: src/views/temp.vue (hunk 3 above)
Comment (alias01) [reply src/views/temp.vue:1]: let it be for a moment...will delete later

File: src/store/index.js (hunk 2 above)
Comment (alias01) [reply src/store/index.js:8]: ok

File: src/router/routes.js (hunk 1 above)
Comment (alias01) [reply src/router/routes.js:131]: will do it later..let it be for a moment

**Review by arrogant-ant — APPROVED** [review]

**Pull request merged**